styles.css

```diff
@@ -110,8 +114,136 @@ a {
   font-size: 0.7rem;
 }
 
-.social_media_links {
+.social_media_links,
+.work ul {
   display: flex;
   gap: 1rem;
   margin-top: 1.2rem;
 }
+
+img {
+  height: 250px;
+  width: 100%;
+  margin-bottom: 0.7rem;
+}
+
+.work h2,
+.contact_us h2 {
+  font-weight: 700;
+  font-size: 2.2em;
+  color: #7d73fc;
+}
+
+.work_one,
+.work_two,
+.work_three,
+.work_four {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  margin: 1rem;
+}
+
+.work {
+  display: grid;
+  place-content: center;
+  background:
+    url("Ellipse_19.png") -30px -110px no-repeat,
+    url("Ellipse_20.png") 1610px 170px no-repeat,
+    url("Ellipse_22.png") 1484px 975px no-repeat,
+    url("Ellipse_21.png") -175px 800px no-repeat;
+  width: 100%;
+  gap: 2.5rem;
+  justify-items: center;
+}
+
+.work p {
+  margin-bottom: 1rem;
+}
+
+.work li {
+  border-right: 2px solid #646369;
+  padding-right: 0.7rem;
+}
+
+.work ul {
+  margin-bottom: 1.5rem;
+}
+
+.btn {
+  font-size: 0.85rem;
+  color: white;
+  background-color: #9390ff;
+  border: none;
+  border-radius: 5px;
```

Comment:
- [ ] kindly remove the border raduis,  to make your button match with the button design in  the figma .
## your button 
![image](https://user-images.githubusercontent.com/95297251/185715830-1f5a0573-3345-4084-b7ba-6ee6e82f084e.png)
## button in figma design
![image](https://user-images.githubusercontent.com/95297251/185715934-26842347-8c2f-4e81-8278-b53cf31ef03c.png)

Reply:
Good job following the advice of the previous reviewer.